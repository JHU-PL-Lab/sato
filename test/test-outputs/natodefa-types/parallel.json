[
  {
    "err_errors": [
      [
        "Error_binop",
        {
          "err_binop_left_aliases": [],
          "err_binop_right_aliases": [],
          "err_binop_left_val": "0",
          "err_binop_right_val": "1",
          "err_binop_operation": "0 == 1"
        }
      ]
    ],
    "err_input_seq": [ 1 ],
    "err_location": "assert (0 == 1)",
    "err_steps": 540
  },
  {
    "err_errors": [
      [
        "Error_value",
        { "err_value_aliases": [], "err_value_val": "false" }
      ]
    ],
    "err_input_seq": [ 0 ],
    "err_location": "assert false",
    "err_steps": 323
  }
]