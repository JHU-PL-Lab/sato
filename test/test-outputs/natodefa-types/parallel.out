Type errors for:
- Input sequence : [0]
- Found at expr  : assert false
--------------------
* Value : false
Type errors for:
- Input sequence : [1]
- Found at expr  : assert (0 == 1)
--------------------
* Constraint : 0 == 1
2 errors found. No further control flows exist.
